supervisor/addons/addon.py

```diff
@@ -58,17 +60,25 @@
 from ..exceptions import (
     AddonConfigurationError,
     AddonsError,
+    AddonsJobError,
     AddonsNotSupportedError,
     ConfigurationFileError,
     DockerError,
     HostAppArmorError,
 )
 from ..hardware.data import Device
 from ..homeassistant.const import WSEvent, WSType
+from ..jobs.decorator import Job
 from ..utils import check_port
 from ..utils.apparmor import adjust_profile
 from ..utils.json import read_json_file, write_json_file
-from .const import WATCHDOG_RETRY_SECONDS, AddonBackupMode
+from .const import (
+    WATCHDOG_MAX_ATTEMPTS,
+    WATCHDOG_RETRY_SECONDS,
+    WATCHDOG_THROTTLE_MAX_CALLS,
+    WATCHDOG_THROTTLE_PERIOD,
```

Comment:
I guess you wanted import that from `addons.const`?

Reply:
Isn't that where they are imported from? That's what I see...

Reply:
Or are you really asking "why isn't everything importing the same constants since they all have the same values"? I can do that I just figured we might want to tweak them different for core vs plugins vs addons now or in the future.